core/templates/dev/head/collection_editor/CollectionEditor.js

```diff
@@ -28,6 +28,64 @@ oppia.constant(
 oppia.constant(
   'EXPLORATION_SUMMARY_DATA_URL_TEMPLATE', '/explorationsummarieshandler/data');
 
+// Sorts the node list for display in collection editor frontend. The list
+// starts with a startNode which contains no prerequisite skills. After that
+// nodes are added to the list if their prerequisite skills match the acquired
+// skills of the last node that was added to the node list.
+oppia.filter('customSorter', function() {
```

Comment:
Since the pre-requisite and acquired skills are pre-populated I think this filter need not to exist at all. The only case where I find the need for this, is when the input is a yaml file. For that case is it possible to have the input just be a list of exploration ids?